case-lib/lib.sh

```diff
@@ -711,7 +709,10 @@ is_zephyr()
 get_firmware_path()
 {
     journalctl_cmd -k |
-        awk '/sof.*request_firmware/ { sub(/^.*request_firmware/,""); last_loaded_file=$1 } END { print last_loaded_file }'
+        awk '/sof.*[[:blank:]]request_firmware[[:blank:]]/ {
+               sub(/^.*request_firmware/,""); last_loaded_file=$1
+             }
+         END { print last_loaded_file }'
```

Comment:
Was this broken before?

Reply:
It was broken on my system that has some extra logs, check the commit message.

Grepping logs is "broken by design". In this case the real fix will be https://github.com/thesofproject/linux/issues/3867 but that's not at the top of my TODO list right now.

A more restrictive regex is always "LESS broken by design" - until someone changes the format of the log message. Unlikely here but possible.